### ISSUE TITLE
while loop in local test

### ISSUE THREAD

**Issue by yuchensun97** [vietnguyen91/Super-mario-bros-A3C-pytorch#14]
Hi, I am wondering why there is no break statement in the local_test in this script [https://github.com/uvipen/Super-mario-bros-A3C-pytorch/blob/master/src/process.py]. It seems like the testing loop will run forever. How to terminate the local test?

**Comment by QasimWani** [comment]
Good point, I think it should've been while done==True or something. This is because done is declared previously. Does this make sense to you?
Thanks.
